### PR TITLE
Only applies to Enterprise Edition

### PR DESCRIPTION
**Pull request by joinimran**
As suggested by the user, the DG readiness is only applied to the enterprise edition of Windows 10. Added this info

Problem:  https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7901

### CODE REVIEW

**Comment by illfated** [comment]
May I ask you to correct the typo (autocorrect?) in the PR title, "addition" — I believe you meant to write 'edition'.

File: windows/security/identity-protection/credential-guard/dg-readiness-tool.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ ms.reviewer:
 
 # Windows Defender Device Guard and Windows Defender Credential Guard hardware readiness tool
 
+**Applies to**
+-   Windows 10 Enterprise Edition
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/dg-readiness-tool.md:21]: 2 details here (maybe cosmetic, but according to Codestyle):
```suggestion
**Applies to:**
- Windows 10 Enterprise Edition
```
1. Colon to signify that something follows, especially for a paragraph title.
2. Only 1 space (blank) is needed between the bullet point dash and the text.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**